### PR TITLE
Feature/transparency baselines

### PR DESCRIPTION
**Pull request by erees1**

### CODE REVIEW

File: cot_transparency/openai_utils/inference.py [hunk 1]
```diff
@@ -229,13 +238,18 @@ def gpt4_rate_limited(config: OpenaiInferenceConfig, messages: list[ChatMessages
     return parse_chat_prompt_response_dict(prompt=messages, response_dict=response_dict)
 
 
+load_dotenv()
```
Comment (thejaminator) [inline cot_transparency/openai_utils/inference.py:241]: why does it need to load env here?

File: cot_transparency/openai_utils/set_key.py [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+def set_opeani_org_from_env_rand():
+    ids = os.getenv("OPENAI_ORG_IDS", None)
+    if ids is None:
+        return
+    ids = ids.split(",")
+    if len(ids) > 1:
+        id_ = random.choice(ids)
+    else:
+        id_ = ids[0]
+    openai.organization = id_
```
Comment (thejaminator) [inline cot_transparency/openai_utils/set_key.py:25]: haha ok

Comment (thejaminator) [reply cot_transparency/openai_utils/set_key.py:25]: typo in func name

File: cot_transparency/openai_utils/inference.py (hunk 1 above)
Comment (thejaminator) [reply cot_transparency/openai_utils/inference.py:241]: if its on purpose, can you write a comment why?

Comment (erees1) [reply cot_transparency/openai_utils/inference.py:241]: Removed because we load at the top

File: cot_transparency/openai_utils/set_key.py (hunk 2 above)
Comment (erees1) [reply cot_transparency/openai_utils/set_key.py:25]: fixed

**Pull request merged**